akka-cluster-tools/build.sbt

```diff
@@ -1,14 +1,8 @@
 import akka.{ AkkaBuild, Dependencies, Formatting, MultiNode, ScaladocNoVerificationOfDiagrams, OSGi }
-import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
-
 Formatting.formatSettings
-
 OSGi.clusterTools
-
 Dependencies.clusterTools
 
-MimaKeys.previousArtifacts := akkaPreviousArtifacts("akka-cluster-tools").value
-
```

Comment:
Yea. I remember that you like them. :) However these empty lines bloat build files quite a bit. Also using new lines sparingly allows grouping logically related settings together like [here](https://github.com/2m/akka/blob/bba0528156bce6ff20ae1ae339b476fe3ba188af/akka-docs/build.sbt).